.github/workflows/publish.yml

```diff
@@ -29,4 +33,11 @@ jobs:
           publish: pnpm run changeset:publish
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          NPM_TOKEN: ${{ secrets.WORKLEAP_NPM_TOKEN }}
\ No newline at end of file
+          NPM_TOKEN: ${{ secrets.WORKLEAP_NPM_TOKEN }}
+      
+      - name: LinearB
+        if: ${{ steps.changesets.outputs.published == 'true' }}
+        uses: ./.github/actions/linearb-main
+        with:
+          environment: release
+          timestamp: ${{ steps.linearb-pre.outputs.timestamp }}
```

Comment:
If we use the reusable workflow, you just add it like this:
https://github.com/gsoft-inc/wl-extensions-mongo/blob/fc39caadab7642621ee8b851b8bdb0a9fb8f0601/.github/workflows/publish.yml#L27C3-L32C21

Reply:
If the timestamp is important (see other comments), I would prefer to edit the [shared step](https://github.com/gsoft-inc/wl-reusable-workflows/blob/main/.github/workflows/linearb-deployment.yml) to add an optional parameter. So, you could avoid duplication.